allauth/usersessions/app_settings.py

```diff
@@ -21,6 +21,20 @@ def TRACK_ACTIVITY(self):
         """
         return self._setting("TRACK_ACTIVITY", False)
 
+    @property
+    def HISTORY_ENABLED(self):
```

Comment:
As per our discussion:

> All in all, if we can keep it limited to only the signals part for allauth, then I think there rest can all be done inside your own project?

At this time I don't want to take more in scope than just the signals. E.g. for history you could that we keep a set, or also include timestamp per IP. Possibly, it should also be exposed to the end user. All in all, I don't want to increase scope, and by just having the signals you are free to do as you see fit in your own project.

Reply:
Yes alright, I will revert these parts